src/executables/util_cs.c

```diff
@@ -3202,16 +3197,17 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   check_applied_info = utility_get_option_bool_value (arg_map, APPLYINFO_APPLIED_INFO_S);
   log_path = utility_get_option_string_value (arg_map, APPLYINFO_COPIED_LOG_PATH_S, 0);
+
```

Comment:
trivial;

Please don't make changes which not related to your fix without any purpose.

Reply:
Thank you. this is my mistake.